src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/Ink/EditingCoordinator.cs

```diff
@@ -91,7 +91,7 @@ internal void ActivateDynamicBehavior(EditingBehavior dynamicBehavior, InputDevi
                     //we pass true for userInitiated because we've simply consulted the InputDevice
                     //(and only StylusDevice or MouseDevice) for the current position of the device
                     InitializeCapture(inputDevice, (IStylusEditing)dynamicBehavior, 
-                        true /*userInitiated*/, false/*Don't reset the RTI*/);
+                        userInitiated: true, false/*Don't reset the RTI*/);
```

Comment:
It appears like you missed one here, `Don't reset the RTI` -> `resetDynamicRenderer`